src/tests/unit/tests/DetailsView/components/generic-dialog.test.tsx

```diff
@@ -69,9 +89,15 @@ describe('GenericDialogTest', () => {
             messageText: 'test message',
             primaryButtonText: 'test primary text',
         };
-
-        const rendered = shallow(<GenericDialog {...props} />);
-        rendered.find(Dialog).prop('onDismiss')();
+        useOriginalReactElements('@fluentui/react', [
+            'Dialog',
+            'DialogFooter',
+            'PrimaryButton',
+            'DefaultButton',
+        ]);
+        const renderResult = render(<GenericDialog {...props} />);
+        const button = renderResult.getByRole('button', { name: /Cancel/i });
+        fireEvent.click(button);
```

Comment:
We can keep this test mostly the same as the original test
```suggestion
        render(<GenericDialog {...props} />);
        getMockComponentClassPropsForCall(Dialog).onDismiss();
```

Reply:
Done, please review on latest commit